### PR TITLE
Add GET API

### PR DESCRIPTION
**Pull request by moqmar**
`GET https://ping.farafin.de/api/` gibt jetzt ein JSON-Objekt für die direkte Ausgabe auf der Farafin-Webseite zurück:

```json
{
  "Türstatus FaRa-Büro": {
    "name": "Türstatus FaRa-Büro",
    "type": "door",
    "description": "Türstatus vom FaRa-Büro (G29-103)",
    "data": "open"
  }
}
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,6 +19,12 @@ const doorData = [
 const cards = new Map();
 const times = new Map();
 
+app.get('/api/', (req, res) => {
+    let cardObjects = [...cards].map(x => x[1]);
+    let result = {};
+	for (let i in cardObjects) result[cardObjects[i].name] = cardObjects[i];
+    res.type("application/json").send(JSON.stringify(result));
```
Comment (johannwagner) [inline index.js]: You can use `res.json(result)` at this point.

**Review by johannwagner — CHANGES_REQUESTED** [review]

**Review by johannwagner — APPROVED** [review]

**Comment by moqmar** [comment]
Oh, thanks, just fixed that. Haven't worked with Express in a while :)

**Comment by johannwagner** [comment]
You have to talk to @fin-ger about deploying. I think, this is currently deployed at a Raspberry Pi at the FaraFin office.

**Comment by moqmar** [comment]
Wow, "just reply to this email" works great :joy:

**Comment by moqmar** [comment]
Thanks. Manuel just moved the yado server to our server Asterix after Obelix crashed in January; the Pi only pushes the data to yado.

I'm currently using a docker build directly from my fork, @fin-ger if you could trigger a build in Docker Hub I can change the image back to the original one. :)

**Pull request merged**